doc/articles/faq.md

```diff
@@ -0,0 +1,268 @@
+```
+
+Keep the version 1.1.1 of Microsoft.Extensions.Logging; latest version of Logging Extensions is starting a new thread and it's not supported in Wasm.
+
+## I created my first solution and ran the uwp project. Then I wanted to test the magic so I changed the "Hello world" text, hit save and... no live update. Am I missing something?
+
+For live update on UWP you need to edit the xaml while the project is debugging, no need for save, it updates on every keystroke (more or less) you need to update to the latest UWP sdk in your project configuration, change target version to latest
+
+## if a 3rd party has some plugins for uwp, would they render on devices?
+
+If they are being recompiled on top of Uno.UI, they could be running yes.
+
+##	I don't see any "Uno.UI App Solution" from File->Project->New ?
+
+1. Install **Uno.UI Solution Template Visual Studio Extension** https://github.com/nventive/Uno/releases
+2. Look for Uno.UI App Solution under Visual C#
+3. if you still havent found it, try and use the Search box
+
+## Will Uno.UI support a single .NET Standard 2.0 binary model ?
+
+Not at the moment. iOS and Android platform support relies on the underlying APIs being visible through class hierarchy for performance reasons. Also, the .NET Standard model is based on binary sharing, which makes very difficult to use platform features without jumping through hoops such as Dependency Injection and Inversion of Control.
+
+## Does Uno.UI support having controls in a class library ?
+
+Yes, here's a [project sample](https://github.com/nventive/Uno.Samples/tree/master/UI/ControlLibrary).
+
+##	How to port an existing UWP app to Uno.UI?
+
+First create a shared project and move all .cs and .xaml files into it and reference the project in your UWP head project.  Ensure everything still working and add other projects for other platforms referencing the same shared project.
+
+1. Create a project with Uno.UI template.
+2. Copying as much code as possible from the existing UWP app to the "My Project.Shared"
+3. Add platform specific code using suffixing files in the shared project (ex: ".iOS.cs")
+4. ios/ android specific heads should be relatively empty, only used os specific implementation (push notification handeling/ deeplinking / permissions)
+5. Test, debug and publish.
+
+# Is it at all possible to make http web requests using the WASM target?
+
+Yes it is possible, but you need to use the provided HttpHandler by Uno.UI like what we did in the Uno.UI Playground:
+
+```csharp
+var handler = new Wasm.WasmHttpHandler();
+var httpClient = new HttpClient(handler);
+```
+
+## How do you choose which APIs are being implemented ?
+
+- If the API is present in .NET Standard, it is generally suggested to use this on instead of the UWP one. (e.g System.IO or System.Net.HttpClient)
+- Based on the popularity of suggestions in the [Uno.UI issues list](https://github.com/nventive/Uno/issues). Make sure to open one for the APIs you need.
```

Comment:
Half of this sentence seems to be missing. 
eg 'Missing APIs ill be implemented based on the popularity of suggestions in the [Uno.UI issues list](https://github.com/nventive/Uno/issues).'

Reply:
It was just a continuation of the title, but it's probably best to say it completely :)